index.html

```diff
@@ -1,14 +1,511 @@
+                - Yes, we're all cousins.
+                
+                  
+                - Right. You're right.
+                - At Honex, we constantly strive
+                
+                  
+                to improve every aspect
+                of bee existence.
+                
+                  
+                These bees are stress-testing
+                a new helmet technology.
+                
+                  
+                - What do you think he makes?
+                - Not enough.
+                </p>
+            </article>
+            
+            <!--Line Break-->
+            <div class="line-break">
+              <hr>
+              <hr>
+            </div>
+
+            <!--Learn More Articles-->
+            <!--Article One-->
+            <article>
+              <h5>WHAT'S UP WITH <span class="bold">BEES</span></h5>
+              <div>
+                <img class="learn-pic" src="Pictures/bee-movie07.jpg">
+                <p> 
+                  Like what? Give me one example.
+                  
+                    
+                  I don't know. But you know
+                  what I'm talking about.
+                  
+                    
+                  Please clear the gate.
+                  Royal Nectar Force on approach.
+                  
+                    
+                  Wait a second. Oheck it out.
+                  
+                    
+                  - Hey, those are Pollen Jocks!
+                  - Wow.
+                  
+                    
+                  I've never seen them this close.
+              </div>
+                <button class="button-pink">
+                  <i class="far fa-file"></i>
+                  <a href="/learn-more.html">LEARN MORE</a></button>
+            </article>
+
+            <hr class="line-break-2">
+
+            <!--Article Two-->
+            <article>
+              <h5>ANGRY <span class="bold">BEE</span></h5>
+              <div>
+                <img class="learn-pic" src="Pictures/bee-movie08.jpg">
+                <p> 
+                  Like what? Give me one example.
+                  
+                    
+                  I don't know. But you know
+                  what I'm talking about.
+                  
+                    
+                  Please clear the gate.
+                  Royal Nectar Force on approach.
+                  
+                    
+                  Wait a second. Oheck it out.
+                  
+                    
+                  - Hey, those are Pollen Jocks!
+                  - Wow.
+                  
+                    
+                  I've never seen them this close.
+              </div>
+                <button class="button-pink">
+                  <i class="far fa-file"></i>
+                  <a href="/learn-more.html">LEARN MORE</a>
+                </button>
+            </article>
+          </div>
+        </div>
+      </section>

```

Comment:
The styling of this whole section is very dependent on grid for it's layout. We looked at your site on two monitors between TA's just to double check. It appears in a narrow window the layout becomes a bit closer to how the remock screenshots are. An alternative solution to trying to re-layout this entire grid area for fine tuning, would be to break some of the smaller components of the page section out and use flex boxes. Grid can be difficult to work with as you well know